client/my-sites/plan-compare-card/style.scss

```diff
@@ -60,12 +60,7 @@
 }
 
 .plan-compare-card-item {
-	display: flex;
```

Comment:
labels and check marks are somewhat misaligned (see screenshot below); although this change seems to affect this (mis-)alignment it's not better or worse, just different. it also seems margins between items are off for some of them.

<img width="1709" alt="Screenshot 2019-05-28 at 12 00 35" src="https://user-images.githubusercontent.com/9202899/58469712-51b1af00-8140-11e9-9b13-dc715bc8d1da.png">

Reply:
@flootr In 6d33327 I simplified the list checkmark markup (removed the extra `span`) and improved the CSS by using `vertical-align: text-bottom`. Let me know if it looks better now! It's still going to be a bit different from the flexbox version before, because the line heights and box alignment are done differently in flexbox and block/inline styles.

Reply:
It looks much better now, thanks @jsnajdr 👍